### ISSUE TITLE
Optional inclusion of a picture in the header

### ISSUE THREAD

**Issue by hkoel01** [posquit0/Awesome-CV#19]
Hi posquit0,
while the CV template is absolutely great, it would be good to have the option of
including a foto in the header. This would help, since this is required for a lot of countries and companies.
Maybe add an alternative header which is more centered to the left and add a picture frame on
the right?
Best regards
hk

**Comment by stesba** [comment]
Hi,
together with posquit0 (thanks for the support!) I think we have managed to find a nice and clean solutions for adding a picture in the header.
Since everyone has a different picture's format this solution could fit you all!
At the beginning of resume.tex add the following lines:

```
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     3rd party packages
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% Needed to divide into several files
\usepackage{import}
\usepackage{wallpaper}
\addtolength{\wpXoffset}{-5.8cm}
\addtolength{\wpYoffset}{12.5cm}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     Personal Data
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% Essentials
\ThisCenterWallPaper{0.20}{yourpicture.png}
\name{Claud D.}{Park}

```
The `\usepackage{wallpaper}` allows you both to scale your picture changing the numerical value in the field `\ThisCenterWallPaper{0.20}{yourpicture.png}` and to adjust its absolute offset within the page via the lines

```
\addtolength{\wpXoffset}{-5.8cm}
\addtolength{\wpYoffset}{12.5cm}
```
Hope this solution can help you!

**Comment by hkoel01** [comment]
It works so far, thank you very much! But I would suggest to make an alternative header with the picture 20% on one side and 80% centered header on the other. I may commit my solution later on, if posquit0 is interested.

**Comment by stesba** [comment]
I would be definitely interested.
Mine was just a quick workaround, having a more weighted header would
definitely be a plus.
Il 22/gen/2016 11:37, "Holger Koelmann" <notifications@github.com> ha
scritto:

**Comment by posquit0** [comment]
@hkoel01 I'm also much interested in alternative headers and photo on resume. Please comment your solution on this thread later. I'll review then.

**Comment by hkoel01** [comment]
No problem, I will do that later, but I lack a bit of time at the moment. Might take a couple of days.

**Comment by stesba** [comment]
Hi @hkoel01 !
I was also trying to set up a 20/80 header but I am not satisfied with the result at all.
Did you manage to get a good result?

**Comment by johannesbottcher** [comment]
https://github.com/johannesbottcher/Awesome-CV/tree/photo

Defines the header to be a tabular with the width of the linewidth. If nothing is specified, a mandatory space of `1em`  is left, which will look good even if the content of the main header is so freaking full. 

The content is aligned to the bottom, which some users will want to change. Well, they should be satisfied to place something.

If you want, i make a PR.

**Comment by posquit0** [comment]
@johannesbottcher , your work is very nice :+1: 
However, I need more time to consider about this feature.

I'll let you know when I make a decision. Thanks to your contributions.

**Comment by SNBasti** [comment]
Is there any possibility to use the propose made by @johannesbottcher as full cv-draft ? At the moment I just obtain the header + photo running make from the git branch. 

Thanks in advance

**Comment by posquit0** [comment]
@SNBasti You can remove `\end{document}` in the middle of template source code.

**Comment by johannesbottcher** [comment]
popped up once more http://tex.stackexchange.com/q/294551/37907

**Comment by nitrogl** [comment]
I've just found this thread... but I've just made a pull request of my "draft" solution a few days ago.
In any case, I hope it'll be of help.

https://github.com/nitrogl/Awesome-CV

**Comment by posquit0** [comment]
Resolved in #55
